client/src/bastionlab/polars/client.py

```diff
@@ -34,6 +45,48 @@ def send_df(
         policy: Policy = DEFAULT_POLICY,
         sanitized_columns: List[str] = [],
     ) -> "FetchableLazyFrame":
+        """
+        This method is used to send `polars.internals.dataframe.frame.DataFrame` to the BastionLab server.
+
+        It readily accepts `polars.internals.dataframe.frame.DataFrame` and also specifies the DataFrame policy and a list of
+        sensitive columns.
+
+        Args
+        ----
+        df : polars.internals.dataframe.frame.DataFrame
+            Polars DataFrame
+        policy : bastionlab.polars.policy.Policy
+            BastionLab Remote DataFrame policy. This specifies which operations can be performed on
+            DataFrames and they specified the _data owner_.
```

Comment:
Why is the term data owner surrounded by _ ?

Reply:
not really necessary, just to specify italics.

Reply:
Please remove it, because this is the only place you are doing this.